### ISSUE TITLE
Unexpected string with @react-email/components module

### ISSUE THREAD

**Issue by AydinTheFirst** [fwouts/previewjs#2282]
### Describe the bug

i get this error 
![image](https://github.com/fwouts/previewjs/assets/84444757/90fa6fbc-7453-40c1-ad46-09a483de4450)

my code

```
import React from "react";
import { Button } from "@react-email/components";

export const verifyEmail = () => {
  return (
    <>
      <Button>asd</Button>
    </>
  );
};

```

### Reproduction

I click preview ?

### Preview.js version

Latest on vsc

### Framework

React

### System Info

```shell
Visual studio code, Windows 11
```

### Used Package Manager

npm

**Comment by fwouts** [comment]
Thank you @AydinTheFirst, the error could definitely be improved here. Will look into it.

**Comment by fwouts** [comment]
Hi @AydinTheFirst, this should now be fixed in v1.28.0 on VS Code :)
